### DISCUSSION TITLE
关于 van-row 的 gutter 问题

### DISCUSSION THREAD

**Discussion by wjw-gavin** [youzan/vant#11934] [General]
### 重现链接

无

### Vant 版本

vant 4

### 描述一下你遇到的问题。

今天又用到了这个组件，说下自己的感受：
1. 我觉得如果只是滚动条的话，可以文档说明下配合父容器来解决或者别的方案，因为我觉得这样用户也更容易解决。而且我查看了比较主流的UI组件，几乎都有存在这个问题...
2. 现在这种计算虽然解决了滚动条的问题，但是在布局上对不齐反而觉得比滚动条带来的影响更大？

![image](https://github.com/youzan/vant/assets/19986739/f32525ca-8858-44bd-8c08-e778f6161ef9)

当我一排有多个，有多排，最后一排又不确定个数的情况下，处理对齐需要借助 `js` 计算了，而如果只是滚动条问题，则容易解决一些...
关联pr https://github.com/youzan/vant/pull/11639

### 重现步骤

无

**Comment by inottn** [comment]
复现链接供讨论：https://codesandbox.io/s/vant-3-issue-template-forked-bxp4k2?file=/src/App.vue

**Comment by chenjiahan** [comment]
负 margin 的问题是 100% 会导致滚动条溢出，这也意味着，如果我们要采取额外的兼容方案，每一个使用 gutter 的用户都需要采用该方案才能避免滚动条溢出问题。

我觉得我们可以先探讨下有什么低成本（最好是使用者不感知）的方式可以解决滚动条溢出，如果有合适的方案，我们可以考虑切换回负 margin 方案的。
